src/index.js

```diff
@@ -84,12 +84,14 @@ const composeAssetClip = ({clipName, ref, start, duration, audioRole = 'dialogue
 const jsonToFCPX = (sequenceEDLJson)=>{
     let counterIdAssetClips = 1;
     // sequenceEDLJson.events 
-    const assetClips = sequenceEDLJson.events.map((event, index)=>{
-        const results = composeAssetClip({
+    const assetClips = sequenceEDLJson.events.map((event, index) => {
+        const durationFrames = (event.endTime - event.startTime) * 2500;
```

Comment:
2500 is something of a 'magic number' here - perhaps declare `const FRAME_RATE = 2500` (or whatever name makes sense) and replace the references to it. Ditto 100 - does that have a particular meaning?

Reply:
great point - will do that!